bot.go

```diff
@@ -96,7 +95,7 @@ func buildParams(in Params) (out url.Values) {
 }
 
 // MakeRequest makes a request to a specific endpoint with our token.
-func (bot *BotAPI) MakeRequest(endpoint string, params Params) (APIResponse, error) {
+func (bot *BotAPI) MakeRequest(endpoint string, params Params) (*APIResponse, error) {
```

Comment:
This smells like a breaking change to me. People running off of master or develop are going to have a bad time.

Reply:
Yeah. Not really sure what else to do though, I want to keep consistency through everything and there's already a ton of breaking changes elsewhere so might as well do it now.